pkg/shed/db.go

```diff
@@ -95,6 +96,7 @@ func NewDBWrap(ldb *leveldb.DB) (db *DB, err error) {
 
 	db = &DB{
 		ldb:     ldb,
+		sch:     nil,
```

Comment:
no need to initialize a field that is a pointer to a `nil` value as this is the default value

Reply:
I included that just to make it clear that the initial state is that no schema is currently cached.

Reply:
it is not needed. a comment should suffice